### DISCUSSION TITLE
can the new discharger mode have a rotation offset effect the work area

### DISCUSSION THREAD

**Discussion by thebikwirm** [scfmod/FS22_TerraFarm#112] [General]
can the new discharger mode have a rotation offset effect the work area
got some side dumping trailers that I want to put the discharge effect on, can this be done?
eg 
![image](https://github.com/user-attachments/assets/0ad4d4f9-511d-4b7c-aa16-12a7d34c8e8e)

**Comment by scfmod** [comment]
Yeah there is a rotation attribute on the workArea entry
https://github.com/scfmod/FS22_TerraFarm/blob/main/docs/MACHINE_DISCHARGER.md#work-area
This should enable you to rotate it, if you have any issues with it let me know
